project/profiles/capacity4more/behat/features/group_dashboard.feature

```diff
@@ -135,3 +135,10 @@ Feature: Group dashboard
     | galileo     |
     | badhairday  |
     | president   |
+
+  @api
+  Scenario: Highlighted document content title should contain a link.
+    Given I am logged in as user "badhairday"
+    When  I visit the dashboard of group "Nobel Prize"
+    Then  I should see the text "Nobel Prize"
+    And I should see the link "Nobel Prize sheet"
```

Comment:
@ordavidil 
After migration, "Nobel Prize sheet" link appears on group dashboard only once, at 'Highlighted' block. Why do we need to access the link? Isn't it enough to make sure we see it's a link.

There's no 'highlight box class'. Adding it to all content types will take time.
Creating 'I visit the highlighted link ..' function will take time.

I don't think this test worth the time that need to be spent.